src/elexmodel/client.py

```diff
@@ -86,6 +88,21 @@ def _check_input_parameters(
             raise ValueError("handle_unreporting must be either `drop` or `zero`")
         return True
 
+    """
```

Comment:
to make docstrings visible when you are calling the function, they should go under the function header. So:
```
def get_conformalization_data_unit(self):
"""
docstring goes here
"""
```

Reply:
done